Userland/Applications/FileManager/PropertiesWindow.cpp

```diff
@@ -226,12 +238,12 @@ bool PropertiesWindow::apply_changes()
     return true;
 }
 
-void PropertiesWindow::setup_permission_checkboxes(GUI::CheckBox& box_read, GUI::CheckBox& box_write, GUI::CheckBox& box_execute, PermissionMasks masks, mode_t mode)
+ErrorOr<void> PropertiesWindow::setup_permission_checkboxes(GUI::CheckBox& box_read, GUI::CheckBox& box_write, GUI::CheckBox& box_execute, PermissionMasks masks, mode_t mode)
 {
     struct stat st;
     if (lstat(m_path.characters(), &st)) {
```

Comment:
same comment about Core::System::lstat being the preferred way to port this function.